tests/helixperftasks.targets

```diff
@@ -0,0 +1,89 @@
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="GetPerfTestAssemblies" AssemblyFile="$(ToolsDir)net45/Microsoft.DotNet.Build.CloudTestTasks.dll"/>
+  <Import Project="helix.targets" />
```

Comment:
That target would go away soon. It was added because of xunitwrappers required for functional runs. Since for perf runs you do not have that problem....it should not be required ideally.